meshtastic/tests/test_smoke1.py

```
@@ -143,535 +191,455 @@ def test_smoke1_qr() -> None:
+    return_value, out = _run("meshtastic --sendtext hello")
+    _assert_connected(out)
+    assert "Sending text message hello" in out
     assert return_value == 0
 
 
 @pytest.mark.smoke1
 def test_smoke1_port() -> None:
-    """Test --port."""
-    # first, get the ports
+    """`--port <detected-port> --info` should connect to the same radio."""
     ports = findPorts(True)
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)